server/request/auth_request.go

```diff
@@ -1,6 +1,16 @@
 package request
 
+import validation "github.com/go-ozzo/ozzo-validation"
+
 type AuthRequest struct {
-	Email    string `json:"email" binding:"required,email" example:"user@example.com"`
-	Password string `json:"password" binding:"required" example:"11111111"`
+	FullName    string `json:"fullName"`
+	Email       string `json:"email"`
+	NewPassword string `json:"newPassword" binding:"required" example:"11111111"`
```

Comment:
what is the significane of `example:"11111111"` here?

Reply:
example:"11111111" is to inform that password should be atleast 8 characters instead it should be like example:P@ssw0rd